File: 01 Introdução/1002 Área do Circulo/1002.java
import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    double pi = 3.14159;

    Scanner input = new Scanner(System.in);
    double raio = input.nextDouble();

    System.out.printf("A=%.4f\n", raio * raio * pi);
    input.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1002
// Ranking 29131º, Time 0.182, Submiss Date 12/02/2023 22:00:22 (Java 14)

File: 01 Introdução/1002 Área do Circulo/1002.cpp
#include <iostream>
#include <iomanip> 

using namespace std;
 
int main() {
 double raio;
 cin >> raio;
 cout << "A=" << fixed << setprecision(4) << raio*raio*3.14159 << endl;
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1002
// Ranking 72760º, Time 0.000, Submiss Date 16/03/2023 23:31:35 (C++17)